src/main/java/com/zufar/onlinestore/user/api/AuthorityService.java

```diff
@@ -0,0 +1,19 @@
+package com.zufar.onlinestore.user.api;
+
+import com.zufar.onlinestore.user.entity.Authority;
+import com.zufar.onlinestore.user.entity.UserEntity;
+import com.zufar.onlinestore.user.entity.UserGrantedAuthority;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AuthorityService {
```

Comment:
`DefaultUserAuthoritySetter` sound clearer

Reply:
I think that this service may have a greater responsibility than setting the default role